pinax/stripe/migrations/0019_account.authorized.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.6 on 2017-10-25 10:41
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('pinax_stripe', '0018_add_discounts'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='account',
+            name='authorized',
+            field=models.BooleanField(default=True),
+        ),
+    ]
```

Comment:
You cannot have a dot in the file basename: `ModuleNotFoundError: No module named 'pinax.stripe.migrations.0019_account'`

Reply:
thx